### ISSUE TITLE
[css-backgrounds][css-logical-props] Define logical values for <side-or-corner>

### ISSUE THREAD

**Issue by jonathanKingston** [w3c/csswg-drafts#974]
In [the linear gradient syntax](https://drafts.csswg.org/css-images-3/#linear-gradient-syntax) there currently isn't the ability to use logical gradient sides.

Can we consider adding the ability to use:

```
background: linear-gradient(to block-end, white, black);
```

Related to: #549

**Comment by fantasai** [comment]
Definitely need to add these. I'd say 100% chance that it oughtta go into css-images-4.

**Comment by LeaVerou** [comment]
@fantasai @tabatkins That section has not yet been copied over to CSS Images 4, since it's still a delta(ish) spec. Is it stable enough to copy now and make this change or should I hold off these edits until Images 3 is more stable?

**Comment by fantasai** [comment]
Yeah, agreed we should wait until Images 3 is republished. It probably needs some cleanup. But we should definitely fix this, maybe add a placeholder issue in Images 4 for now?

**Comment by GoldenFalls** [comment]
Does anyone know the status of this issue? In the [CSS Images Module 3 draft section about linear gradients](https://drafts.csswg.org/css-images-3/#linear-gradient-syntax) there's a note that "next level of this module will provide the ability to define the gradient’s direction relative to the current text direction and writing-mode." However, the [CSS Images Module 4 draft section](https://drafts.csswg.org/css-images-4/#linear-gradients) just says there's no changes, and there's not even a note about flow-relative syntax anymore.

I've never made any comments on the whole process of developing CSS before, so forgive me if I'm missing something. I was just really surprised when I tried to use `to inline-end` in my linear gradient and it said that was invalid, so I went investigating and found this. Gave this issue a thumbs up, really hope this change gets implemented!
